public/index.html

```diff
@@ -57,9 +58,10 @@
         font-weight: 535;
         font-style: normal;
         font-display: block;
-        font-named-instance: 'Book';
+        font-named-instance: 'Medium';
```

Comment:
What is the difference between `Medium` and `Book`? Are they *both* in use?

Reply:
It looks like these instance names are never actually referenced, but these *are* disambiguated by their font weights, and selected via sprinkles' fontWeight vars.